superset-frontend/src/SqlLab/components/ExploreResultsButton.jsx

```diff
@@ -152,14 +152,11 @@ class ExploreResultsButton extends React.PureComponent {
   }
 
   render() {
-    const allowsSubquery =
-      this.props.database && this.props.database.allows_subquery;
```

Comment:
Hugh, why are we removing this? In databases that don't support subqueries we can't show the button because Explore will fail.